### PR TITLE
OCPQE-12897: Integrate ocp tests in opp

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

File: ci-operator/config/stolostron/policy-collection/stolostron-policy-collection-main__opp-ocp4.12-interop.yaml [hunk 1]
```diff
@@ -50,6 +50,7 @@ tests:
     - ref: ipi-install-monitoringpvc
     - chain: ipi-install
     test:
+    - chain: cucushift-installer-check-cluster-health
```
Comment (gparvin) [inline ci-operator/config/stolostron/policy-collection/stolostron-policy-collection-main__opp-ocp4.12-interop.yaml:53]: Can you move this line after the line:
```
- ref: acm-policies-openshift-plus
```

That way it runs the test with OPP already installed.  Thanks

**Comment by liangxia** [comment]
The test failed with error "Error policies failed to become compliant in allotted time."
Retest.

/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.12-interop-openshift-plus-interop-aws

**Pull request merged**